test-case/check-pause-release-suspend-resume.sh

```diff
@@ -75,21 +70,21 @@ OPT_NAME['l']='loop'         OPT_DESC['l']='loop count'
 OPT_HAS_ARG['l']=1             OPT_VAL['l']=5
 
 OPT_NAME['i']='sleep-period' OPT_DESC['i']='sleep period of aplay, unit is ms'
-OPT_HAS_ARG['i']=1             OPT_VAL['i']='100'
+OPT_HAS_ARG['i']=1             OPT_VAL['i']='200'
```

Comment:
I've just tested this in my TGLU_SKU0A32_SDCA device and found that 200ms seems still too short for my device while 300ms will work fine. Maybe we need more investigation to determine a proper period, or just left it unchanged and specify a proper period for each device in our CI?

Reply:
How about something much longer but still quick like 1s?